src/component/inspect-request/request-headers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const {object} = React.PropTypes;
+
+const RequestHeaders = (props) => {
```

Comment:
Let's do some `RequestDetails` action for this PR, it sounds pretty nice

On Tue, Feb 9, 2016 at 7:26 PM, Tomi <notifications@github.com> wrote:

> In src/component/inspect-request/request-headers.js
> <https://github.com/james-proxy/james/pull/105#discussion_r52350716>:
>
> > @@ -0,0 +1,21 @@
> > +import React from 'react';
> > +
> > +const {object} = React.PropTypes;
> > +
> > +const RequestHeaders = (props) => {
>
> Yeah, KeyValue did hurt a bit. At the time I named this, this was before
> you added the query params feature, and then I discovered it upon rebasing.
> I kind of justified it to myself by saying you could *maybe* consider
> query parameters to be a request header. It's a bit flimsy logic, but it
> kinda works.
>
> Maybe RequestDetails? I'm open to suggestions. Won't hurt to rename later
> either.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/james-proxy/james/pull/105/files#r52350716>.
>